CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# How to use this repo
+
+Please [open Issues](https://github.com/sensu-plugins/sensu-plugins-feature-requests/issues) for anything on your mind that will help the community succeed at having **the best collection of monitoring plugins.**
+
+Common requests are:
+
+* Transfer ownership of your rad Sensu plugin to this organization ([example][1])
+* Offer an awesome idea that will help others make great plugins ([example][3])
+
+If you're ever not sure about how you can help, reach out on [our Community Slack](https://sensucommunityslack.herokuapp.com/)!
```

Comment:
Ya I feel you, honestly I never noticed the sign in link until you pointed it out is the only reason to but I completely understand your point...Maybe we hack their code to make it more noticeable?